### ISSUE TITLE
How to disable registration?

### ISSUE THREAD

**Issue by RPWouters** [lenis0012/LoginSecurity#315]
I want to shutdown the registration possibility after my friends registered. Because now unwanted players can join and register.

**Comment by lenis0012** [comment]
@RPWouters I just added this in the latest dev build. Please give it a tryl
There should be an option to disable registration in the config.

**Comment by lenis0012** [comment]
https://ci.codemc.io/view/Author/job/lenis0012/job/LoginSecurity/133/

**Comment by mohaidoss** [comment]
It's working great thanks ! 
Only struggle is the key names used in the config, having multiple _**enabled**_ keys in the yml, makes it hard to edit the config using `sed` only

**Comment by RPWouters** [comment]
It's indeed working, thank you!;

**Comment by lenis0012** [comment]
> It's working great, thanks @lenis0012 ! Only struggle is the key names used in the config, having multiple _**enabled**_ keys in the yml, makes it hard to edit the config using `sed` only

what would you suggest instead?

**Comment by mohaidoss** [comment]
> > It's working great, thanks @lenis0012 ! Only struggle is the key names used in the config, having multiple _**enabled**_ keys in the yml, makes it hard to edit the config using `sed` only
> 
> what would you suggest instead?

@lenis0012 Just to keep the same structure, I would suggest prefixing the keys. 
For example:
**config.yml before**
```yaml
registration:
  enabled: true
```
**config.yml after**
```yaml
registration:
  registration_enabled: true
```

This config can easily be edited with bash command `sed`, cause there is only one exact match
```bash
sed -i "s|registration_enabled\:*|registration_enabled:false|" ./config.yml```

**Comment by lenis0012** [comment]
> > > It's working great, thanks @lenis0012 ! Only struggle is the key names used in the config, having multiple _**enabled**_ keys in the yml, makes it hard to edit the config using `sed` only
> > 
> > 
> > what would you suggest instead?
> 
> @lenis0012 Just to keep the same structure, I would suggest prefixing the keys. For example: **config.yml before**
> 
> ```yaml
> registration:
>   enabled: true
> ```
> 
> **config.yml after**
> 
> ```yaml
> registration:
>   registration_enabled: true
> ```
> 
> This config can easily be edited with bash command `sed`, cause there is only one exact match
> 
> ```shell
> sed -i "s|registration_enabled\:*|registration_enabled:false|" ./config.yml```
> ```

I see. if that is the issue would something like "accept-new-users" work as well?

**Comment by lenis0012** [comment]
Also I just thought of this but why didn't you just enable whitelisting? @RPWouters

**Comment by mohaidoss** [comment]
> > > > It's working great, thanks @lenis0012 ! Only struggle is the key names used in the config, having multiple _**enabled**_ keys in the yml, makes it hard to edit the config using `sed` only
> > > 
> > > 
> > > what would you suggest instead?
> > 
> > 
> > @lenis0012 Just to keep the same structure, I would suggest prefixing the keys. For example: **config.yml before**
> > ```yaml
> > registration:
> >   enabled: true
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > **config.yml after**
> > ```yaml
> > registration:
> >   registration_enabled: true
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > This config can easily be edited with bash command `sed`, cause there is only one exact match
> > ```shell
> > sed -i "s|registration_enabled\:*|registration_enabled:false|" ./config.yml```
> > ```
> 
> I see. if that is the issue would something like "accept-new-users" work as well?

Yes, it would. But the other config values may need that as well !
